File: openapi-client-old/javascript-closure-angular/API/Client/ErrorResponse.js
goog.provide('API.Client.ErrorResponse');

/**
 * @record
 */
API.Client.ErrorResponse = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.ErrorResponse.prototype.code;

/**
 * @type {!string}
 * @export
 */
API.Client.ErrorResponse.prototype.message;

File: openapi-client-old/scala-gatling/src/gatling/scala/org/openapitools/client/model/ErrorResponse.scala

package org.openapitools.client.model


case class ErrorResponse (
    _code: String,
    _message: String
)
object ErrorResponse {
    def toStringBody(var_code: Object, var_message: Object) =
        s"""
        | {
        | "code":$var_code,"message":$var_message
        | }
        """.stripMargin
}
